criteria.md

```diff
@@ -1,47 +1,83 @@
+# Open Source Project Security Baseline
+
+This document contains the criteria for the baseline. Where applicable, terms must be interpreted according to the definitions in the [OSPS Lexicon](./lexicon.md)
+
+## Criteria
+
 ### Maturity Level 1 (Sandbox)
  
 | Category | Criteria Statement |
 |---|---|
-| Access Control | The project's version control system MUST require multi-factor authentication (MFA) for users when modifying the project repository settings or accessing sensitive data. |
-| Access Control | The project's version control system MUST restrict collaborator permissions to the lowest available privileges by default. |
-| Access Control | The project's version control system MUST prevent accidental direct commits against the primary branch. |
-| Access Control | The project's version control system MUST prevent accidental deletion of the primary branch. |
-| Build & Release | The project's build and release pipelines MUST NOT execute arbitrary code that is input from outside of the build script. |
-| Build & Release | All software releases MUST be assigned a standardized unique version identifier. |
+| Access Control | The project's [version control system] MUST require [multi-factor authentication] for users when modifying the project [repository] settings or accessing sensitive data. |
+| Access Control | The project's [version control system] MUST restrict [collaborator] permissions to the lowest available privileges by default. |
+| Access Control | The project's [version control system] MUST prevent accidental direct [commits] against the [primary branch]. |
+| Access Control | The project's [version control system] MUST prevent accidental deletion of the [primary branch]. |
+| Build & Release | The project's [build and release pipelines] MUST NOT execute [arbitrary code] that is input from outside of the build script. |
+| Build & Release | All [releases] and [released software assets] MUST be assigned a standardized [unique version identifier]. |
```

Comment:
I think "standarized" here needs a little bit more context. Does this mean keeping version labels consistent throughout the project's releases? Using a well known versioning schema?